xmtp-inbox-ios/Loaders/MessageLoader.swift

```diff
@@ -53,10 +55,10 @@ class MessageLoader: ObservableObject {
 	func fetchRemote() async throws {
 		for topic in conversation.topics() {
 			do {
-				let messages = try await topic.toXMTP(client: client).messages(limit: fetchLimit)
```

Comment:
Why the removal of the `limit` here?

Reply:
It's temporary until https://github.com/xmtp/xmtp-ios/pull/70 lands.

Reply:
Now that the remote attachment SDK PR is merged, I could point SPM to the fix-fake-api-pagination branch, so I was able to bring back the limits in https://github.com/xmtp-labs/xmtp-inbox-ios/pull/104/commits/f4c69f2c0525b7ea655ae25bf985ba926c2480f2.